stdlib/http/path_encode_test.go

```diff
@@ -0,0 +1,32 @@
+package http_test
+
+import (
+	"testing"
+
+	"github.com/influxdata/flux/interpreter"
+	"github.com/influxdata/flux/stdlib/http"
+	"github.com/influxdata/flux/values"
+)
+
+// some characters
+// want those escaped
```

Comment:
I think you can drop these comments, no?

Reply:
Yes, will do.